### PR TITLE
adding get-from-bendo

### PR DESCRIPTION
**Pull request by msuhovec**
#36

### CODE REVIEW

File: tasks/fedora-to-rof [hunk 1]
```diff
@@ -16,7 +16,7 @@ if [ ! -e "$pid_file" ]; then
 fi
 
 # Iterate through pids- use rof fedor_to_rof to retrieve from fedora and generate rof
-for pid  in $(jq --raw-output '.[]' < $pid_file); do
+for pid  in $(jq 'keys' < $pid_file | jq --raw-output '.[]'); do
```
Comment (dbrower) [inline tasks/fedora-to-rof]: It seems like there must be some way to combine these two `jq` commands into one. Maybe `jq --raw-output 'keys|.[]'`?

File: tasks/get-from-bendo [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	mkdir -p "${JOBPATH}/FROM_TAPE" 
+fi
+
+# Iterate through pids- use rof blicleint to retrieve from bendo and bendo rof under FROM_TAPE
+for pid  in $(jq 'keys' < $pid_file | jq --raw-output '.[]'); do
+
+  this_pid=$( echo $pid | sed 's/und://')
+  this_bendo=$(jq --raw-output '.[]["bendo-item"]' "${JOBPATH}/${this_pid}.rof")
+
+  if ! /opt/batchs/bin/bclient -root "${JOBPATH}/FROM_TAPE" -token $bclient_api_key -server $bendo_url  get $this_bendo "fedora3/${this_pid}.rof"  ; then
```
Comment (dbrower) [inline tasks/get-from-bendo]: do we use an absolute path for bclient in the other tasks?

File: tasks/fedora-to-rof (hunk 1 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: Probably, but this works, and it's still one line.

File: tasks/get-from-bendo (hunk 2 above)
Comment (msuhovec) [inline tasks/get-from-bendo]: I will change this

File: tasks/fedora-to-rof (hunk 1 above)
Comment (msuhovec) [inline tasks/fedora-to-rof]: that worked- I must have done something with whitespace that broke it when
I tried. You have the majic suace.

I waded through about a dozen jq questions on stackoverfow yesterday, and
'shit - I swore I tried that' was the standard, accepted jq disclaimer.

Comment (dbrower) [inline tasks/fedora-to-rof]: Are you sure this is doing what you want it to do? I tried this and it doesn't seem to work:

```
$ echo '{"keys":[1,2,3]}' | jq 'keys'
[
  "keys"
]
```

Comment (msuhovec) [inline tasks/fedora-to-rof]: The pids hash is different from how you've iimagined it:

cat fedora-pids
{"und:dev00149w5f":"dev00149w5f","und:dev00149w74":"dev00149w5f","und:dev00149w9t":"dev00149w5f","und:dev00149x01":"dev00149w5f","und:dev00149x2q":"dev00149w5f"}

jq --raw-output 'keys|.[]' < fedora-pids

und:dev00149w5f
und:dev00149w74
und:dev00149w9t
und:dev00149x01
und:dev00149x2q

Comment (dbrower) [inline tasks/fedora-to-rof]: ah. thanks. I see now, keys is a function.

**Comment by dbrower** [comment]
Per our conversation, make a task that then starts these new tasks in the proper order. That way we can change the workflow by editing task files instead of the rake code.

**Comment by msuhovec** [comment]
Did that yesterday- start-reingest

**Pull request merged**